docs/guides/hedgehog-engine/blueblur/levels/importing-collision.md

```diff
@@ -1,48 +1,47 @@
 !!! info
     This guide uses the following tools:
 
     - [Sonic GLvl](/tools/hedgehog-engine/blueblur/levels/#sonicglvl){ target="_blank"}
-	- [Blender](/tools/hedgehog-engine/blueblur/levels/#blender){ target="_blank"}
+	- [Blender](https://blender.org){ target="_blank"}
 	- [HedgeArcPack](/tools/hedgehog-engine/common/files/#hedgearcpack){ target="_blank"}
 
-#### Part 1 (Creating collision in Blender)
+#### Part 1 (Creating Collision in Blender)
 You'll need model for the collision. If you want collision with different parameters (or tags), you'll need special ini file for HavokConverter with tags. You can download it [here](assets/importing-collision/HavokConverter.ini).
 Let me show you how to apply tags in your collision: 
 
 First, you need to find mesh on which you want to apply tags. 
 
-Second, you need to add the tag in the end of the name. (for example: level_stone<code style="color: red;">@STONE</code>). 
+Second, you need to add the tag in the end of the name. (for example: level_stone`@STONE`). 
 
-![Tag Example](assets/importing-collision/tag_example.png)
+!!! info "Example"
+    ![Tag example](assets/importing-collision/tag_example.png)
 
-You can find other tags in HavokConverter if you drop this ini file in <code style="color: green;">"libgens-sonicglvl-master/bin"</code>.
+You can find other tags in `HavokConverter` if you drop this ini file in `"libgens-sonicglvl-master/bin"`.
 
-#### Part 2 (Importing your custom collision in the game)
-Open "libgens-sonicglvl-master/bin" folder and drop special ini file in it. Open "HavokConverter.exe" and fill:
+#### Part 2 (Importing Your Custom Collision in the Game)
+Open `libgens-sonicglvl-master/bin` folder and drop special ini file in it. Open `HavokConverter.exe` and fill:
 
-"Source 3D model files"
+- Source 3D model files - Your collision model files
+- Output file - Directory which will contain the converted collision (add phy.hkx in the end of the name)
 
-"Output file" (add phy.hkx in the end of the name)
+![Havok Converter Settings](assets/importing-collision/havok_converter_window.png)
 
+Now you can click `Convert` button and wait. 
 
-![Havok converter settings](assets/importing-collision/havok_converter_window.png)
-
-Next, you'll need to unpack original level's file which is located in the root of .cpk file (for example: <code style="color: green;">#ghz200.ar.00</code>). Unpack it with HedgeArcPack, and drop your collision file. Open <code style="color: green;">"Terrain.stg.xml"</code>, find RigidBodyContainer, and paste here your collision's name without the extension and save it.
+Next, you'll need to unpack original level's file which is located in the root of .cpk file (for example: <code style="color: green;">#ghz200.ar.00</code>). Unpack it with `HedgeArcPack`, and drop your collision file. Open <code style="color: green;">"Terrain.stg.xml"</code>, find `RigidBodyContainer`, and paste here your collision's name without the extension and save it.
```

Comment:
Probably a good idea to replace the HTML tags in this sentence with code blocks:

```
`Terrain.stg.xml`
`#ghz200.ar.00`
```